src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,100 @@
+package virtual_pet;
+
+import java.util.Scanner;
+
+public class VirtualPet {
+    Scanner input = new Scanner(System.in);
```

Comment:
Scanner should not be in VirtualPet class.  All in put should strictly take place in VirtualPetApplication